yt_dlp/extractor/youtube.py

```diff
@@ -3701,6 +3719,53 @@ def _generate_player_context(cls, sts=None):
             **cls._get_checkok_params(),
         }
 
+    def _get_config_po_token(self, client):
+        po_token_strs = self._configuration_arg('po_token', [], ie_key=YoutubeIE, casesense=True)
+        for token_str in po_token_strs:
+            po_token_client, sep, po_token = token_str.partition('+')
+            if not sep:
+                self.report_warning('Invalid po_token configuration format. Expected "client+po_token"')
```

Comment:
Could include the bad arg with an only_once
```suggestion
                self.report_warning(
                    f'Invalid po_token configuration format. Expected "client+po_token", got "{token_str}"', only_once=True)
```